plugins/obs-filters/data/locale/en-US.ini

```diff
@@ -73,3 +73,60 @@ Compressor.Threshold="Threshold (dB)"
 Compressor.AttackTime="Attack (ms)"
 Compressor.ReleaseTime="Release (ms)"
 Compressor.OutputGain="Output Gain (dB)"
+Sepia.Tone="Sepia Tone"
```

Comment:
I asked Jim about it and he said that it is used for context (as you suggested). Fixing now.